firmware/hw_layer/stepper_dual_hbridge.cpp

```diff
@@ -22,12 +22,26 @@ static const int8_t phaseB[] =
     -1
 };
 
+static const int8_t microPhases[] = {
+    0,    20,  38,  56,  71,  83,  92,  98,
+    100,  98,  92,  83,  71,  56,  38,  20,
+    0,   -20, -38, -56, -71, -83, -92, -98,
+   -100, -98, -92, -83, -71, -56, -38, -20
+};
+
+static const int maxNumSteps = 8;
```

Comment:
these can be `constexpr` instead of `const` to force computation at compile time